Remora.Rest/Json/DataObjectConverter.cs

```diff
@@ -733,6 +576,39 @@ private string GetWriteJsonPropertyName(PropertyInfo dtoProperty, JsonSerializer
         return createdConverter;
     }
 
+    /// <summary>
+    /// Gets the default value for a type or <see langword="null"/> if it has no default value.
+    /// </summary>
+    /// <remarks>
+    /// This always either <see langword="default"/> for <see cref="Optional{TValue}"/> or <see langword="null"/> for any other type.
+    /// </remarks>
+    /// <param name="type">The type to get the default value for.</param>
+    /// <returns>The default value or <see langword="null"/>.</returns>
+    internal object? GetDefaultValueForType(Type type)
+    {
+        return _dtoEmptyOptionals.GetValueOrDefault(type);
+    }
+
+    /// <summary>
+    /// Returns whether the specified property is to be included when serializing even if it is read-only.
+    /// </summary>
+    /// <param name="property">The property.</param>
+    /// <returns>Whether the property is to be included even if it is read-only.</returns>
+    internal bool IncludesReadOnlyProperty(PropertyInfo property)
```

Comment:
`ShouldIncludeReadOnlyProperty` might be a more appropriate name for this method.